db/connection.go

```diff
@@ -0,0 +1,43 @@
+package db
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
```

Comment:
Right, this would end up running the init method which should not be done if we are using a non-postgres database.